sources/introduction.asciidoc

```diff
@@ -7,7 +7,7 @@
 
 == Introduction
 
-This document is the specification of the Java API for Java{spec-name-bv} in {spec-name-eeplatform} and Java SE. The technical objective of this work is to provide an object level constraint declaration and validation facility for the Java application developer, as well as a constraint metadata repository and query API.
+This document is the specification of the Java API for JavaBean validation in {spec-name-eeplatform} and Java SE. The technical objective of this work is to provide an object level constraint declaration and validation facility for the Java application developer, as well as a constraint metadata repository and query API.
```

Comment:
This one fixes references to JavaBean. It should be squashed as a fixup commit.

Reply:
oh man, I looked at this one for ever, I knew I had made a mistake like that but could not spot it. Drove  me nuts.